app/views/users/new.html.erb

```diff
@@ -0,0 +1,28 @@
+<% if false %>
```

Comment:
This is a neat way of not showing these forms. Even though the instructions say to comment it out. I'd accept this ✔️ .

Very clever 😄